src/main/resources/csv/restore-cases-sscssi-173.csv

```diff
@@ -0,0 +1,2 @@
+IDs
+1714032555601994
```

Comment:
Note this is the ID of a case in preview so that it can be "restored". Should this be removed after testing?